dart/biomechanics/DynamicsFitter.hpp

```diff
@@ -1254,6 +1254,10 @@ class DynamicsFitter
       std::vector<int> framesPerSecond,
       std::vector<std::vector<std::map<std::string, Eigen::Vector3s>>>
           markerObservationTrials,
+      /// This argument takes a list over trials, where each trial is a list of
+      /// integers, one-per-force-plate, where each integer is the index of the
+      /// GRF node to assign that force plate to, or -1 to just rely on the
+      /// automated assignment algorithm
```

Comment:
Sorry to be nit-picky, but I'm still not sure how I would construct this vector given some ground reaction force data. What does the "index of the GRF node" mean here? Maybe an example would help.